### ISSUE TITLE
Running opt_design produces errors on vivado v2020.2

### ISSUE THREAD

**Issue by Kazhuu** [embed-me/ebaz4205_fpga#1]
Hello! Love your blog and content you produce! I tried to using Vivado project with v2020.2 version. I'm not sure is that the culprit or missing files in the project but synthesis fails with following errors
```
...

Starting DRC Task
INFO: [DRC 23-27] Running DRC with 4 threads
ERROR: [DRC INBB-3] Black Box Instances: Cell 'system_wrapper_i/system_i' of type 'system' has undefined contents and is considered a black box.  The contents of this cell must be defined for opt_design to complete successfully.
WARNING: [DRC RPBF-3] IO port buffering is incomplete: Device port ddr_addr_io[0] expects both input and output buffering but the buffers are incomplete.
WARNING: [DRC RPBF-3] IO port buffering is incomplete: Device port ddr_addr_io[10] expects both input and output buffering but the buffers are incomplete.
...
INFO: [Project 1-461] DRC finished with 1 Errors, 130 Warnings
INFO: [Project 1-462] Please refer to the DRC report (report_drc) for more information.
ERROR: [Vivado_Tcl 4-78] Error(s) found during DRC. Opt_design not run.

Time (s): cpu = 00:00:02 ; elapsed = 00:00:00.87 . Memory (MB): peak = 2334.043 ; gain = 31.754 ; free physical = 7005 ; free virtual = 12020
INFO: [Common 17-83] Releasing license: Implementation
12 Infos, 130 Warnings, 1 Critical Warnings and 2 Errors encountered.
opt_design failed
ERROR: [Common 17-39] 'opt_design' failed due to earlier errors.
...
```
And when trying to open the design with Vivado the file looks like this
![image](https://user-images.githubusercontent.com/11016113/104092052-19fe6900-528a-11eb-8ec2-679975438409.png)

Tried both 2020.1 and main branches and the same result. I new to Vivado tools and FPGA design in general so sorry if this is question that should be obvious.

**Comment by embed-me** [comment]
Hello and thank you!
As you have already seen from the available branches, currently only Vivado Versions up to 2020.1 are supported, so either you fix the issue yourself, or you will have to wait on the 2020.2 update.

**Comment by Kazhuu** [comment]
Thank you for making the 2020.2 release. I'd have done it myself and made a PR too but new to this stuff at the moment :)

**Comment by Applepi** [comment]
Any idea when this will be fixed? It's not clear to me what is different in the 2020.2 branch to fix the blackbox issue.

**Comment by embed-me** [comment]
The issue was fixed with commit 2714528197c0d1de27ef05cae44237f6ea0adcbc. A clean build works for me. If you are still having problems consider opening up a new issue.
